parser/src/main/java/io/specmesh/apiparser/model/SchemaInfo.java

```diff
@@ -14,6 +17,20 @@ public class SchemaInfo {
     private final String contentType;
     private String schemaLookupStrategy;
 
+    /**
+     * @param schemaRef
+     *            location of schema
+     * @param schemaFormat
+     *            format of schema
+     * @param schemaIdLocation
+     *            ???
```

Comment:
part of the async api spec - can be header or payload

Reply:
As agreed, you'll fix forward on these two.